app/assets/javascripts/urlMapHash.js

```diff
@@ -0,0 +1,53 @@
+function urlMapHash() {
+  // This is based off of jywarren's urlhash, made specific to our map hash needs
+
```

Comment:
Hey this is cool, can you tell what specific changes you did to match plots2? Is the URL different now? Thanks!

Reply:
I changed the format to be `#zoom/lat/lon`.  I really like having a hash function available to get and set the values!

Reply:
Quick question, have the layer names been removed from the hash?

Reply:
Currently yes they aren't in the hash. It had been set up to use the route for the layer names, which had to be changed. Do you need them added back in? It's easy enough to add them at the end of the hash, if that works for @jywarren ?

Reply:
No, that's fine for now. I was trying to figure out an issue when I noticed we don't have them anymore. :)

Reply:
We had them to track active layers. In LEL it is used in the embed to preserve the state of the layers. I don't remember if this was the case in plots2? @sagarpreet-chadha, could you confirm?

Reply:
Hey,
So i made embed feature to take array of layers as input, and show them on map by default (@nstjean ...we will use this in /people map in plots2). This feature is unrelated to url hash AFAIK.

If we will require layer names in future, it should be pretty easy, right?  Thanks!

Reply:
Okay. Got it. To correct what I said earlier, it wasn't in the hash but before it. Thanks, @sagarpreet-chadha!